Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Commands/Commands.cs

```diff
@@ -21,6 +21,43 @@ public static Command SelectActionMap(string actionMapName)
             return (in InputActionsEditorState state) => state.SelectActionMap(actionMapName);
         }
 
+        public static Command AddActionMap()
+        {
+            return (in InputActionsEditorState state) =>
+            {
+                var newMap = InputActionSerializationHelpers.AddActionMap(state.serializedObject);
+                var actionProperty = InputActionSerializationHelpers.AddAction(newMap);
+                InputActionSerializationHelpers.AddBinding(actionProperty, newMap);
+                state.serializedObject.ApplyModifiedProperties();
+                return state.SelectActionMap(newMap.FindPropertyRelative(nameof(InputActionMap.m_Name)).stringValue);
```

Comment:
I wonder if SelectActionMap should take an override that takes an action map as it might avoid some string look up at the lower end